web/beacon-app/cypress/e2e/HomePage/HomePage.ts

```diff
@@ -1,232 +1,211 @@
 import { Given, Then, When, And } from 'cypress-cucumber-preprocessor/steps';
 
-Given('I am on the Beacon homepage', () => {
-  cy.loginWith( { email: 'test7@test.com', password:'Abc123Def$56'})
+beforeEach(function () {
```

Comment:
This `beforeEach` function was added so that fixture data may be used.

Reply:
Makes sense!